Sphinx/README.md

```diff
@@ -0,0 +1,19 @@
+Missing:
+cmusphinx-en-us-5.2
+en-us.lm
+
+Get from this link:
+
+https://onedrive.live.com/redir?resid=427D6D69FE5B4BA3!40133&authkey=!AFIK0llMvpxykPk&ithint=file%2czip
+
+Or get files from official source and rename:
```

Comment:
Let's just include instructions on how to get them from official source. What do you mean by rename? Can you make the instructions a little bit more clear?

Do they need to be renamed to `cmusphinx-en-us-5.2` and `en-us.lm`? If so, let's change the first name to `cmusphinx-en-us`.